### PR TITLE
add codeowners

### PR DESCRIPTION
**Pull request by ebrehault**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*       @nuclia/application
+*       @nuclia/lead
```
Comment (operramon) [inline CODEOWNERS:2]: Github compains about `@nuclia/lead`

**Review by operramon — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (vangheem) [reply CODEOWNERS:2]: where is it complaining?

It seems right 

Screenshot 2023-12-20 at 12 53 44

Comment (operramon) [reply CODEOWNERS:2]: when reviewing changed files, I saw this:
Captura de Pantalla 2023-12-20 a les 18 57 47

Comment (ebrehault) [reply CODEOWNERS:2]: right
Nathan has just fixed it

**Pull request merged**